### PR TITLE
Create desktop version

### PR DESCRIPTION
**Pull request by eng-mohamed-eldeeb**
Create desktop version.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -20,7 +20,7 @@ li {
 }
 
 .hero {
```
Comment (rustam-tolipov) [inline style.css:22]: AWESOME DESIGN MOHAMED 👏
- [ ] Your project looks beautiful however there are some requirements and if you follow as I explained you should be approved by the next review. First of all inside `.hero` class please add this line of code and your background image issue will be fixed. 
``` css
background-size: cover; // 👌
````

#### Your design
image

#### After adding that code your design
image

File: style.css [hunk 2]
```diff
@@ -310,6 +318,7 @@ button:focus {
 }
 
 .skills-content {
+  padding-top: 20px;
```
Comment (rustam-tolipov) [inline style.css:322]: - [ ] The second and last thing is in your language cards on the right side of your about section 😀. Please make cards in a row which are you set in the column, ok? 🍀

#### Your design



#### Figma design
image

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @eng-mohamed-eldeeb  , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi @eng-mohamed-eldeeb , 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**